Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1669985965497)
+++ b/.idea/modules.xml	(date 1669985965497)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/proizvodna.iml" filepath="$PROJECT_DIR$/proizvodna.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: proizvodna.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/proizvodna.iml b/proizvodna.iml
new file mode 100644
--- /dev/null	(date 1669985960082)
+++ b/proizvodna.iml	(date 1669985960082)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1669985965623)
+++ b/.idea/misc.xml	(date 1669985965623)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_18" default="true" project-jdk-name="18" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1669986037957)
+++ b/.idea/vcs.xml	(date 1669986037957)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1669985977903)
+++ b/.idea/.gitignore	(date 1669985977903)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
new file mode 100644
--- /dev/null	(date 1669991758364)
+++ b/src/Main.java	(date 1669991758364)
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Это прога придумана для чисел,чтоб продолжит напиши что-то");
+
+        String danger = input.nextLine();
+        System.out.println("Еще раз тебе говорю,не пиши буквы,не будет это не работь");
+        String dangerSecond = input.nextLine();
+        System.out.println("ПРОГРАМА ТЕБЕ ВЫДАСТ ОШИБКУ,я тебя предупредил");
+        String dangerTwo = input.nextLine();
+
+        while (true) {
+            System.out.println("cтепень");
+            System.out.print(">>> ");
+            try {
+                return;
+                int n= input.nextInt();
+            } catch (Exception ex) {
+                System.out.println("Должно быть целое число");
+                System.out.println("Вот наш прикол");
+                double prozv = n - 1;
+
+                System.out.println(n + "x^" + prozv);
+            }
+        /*
+        x^6 -> 6x^5
+         */
+        }
+    }
+}
